### ISSUE TITLE
After running script ‘examples.wmt_2020.en_de.trans_quest’, I got the wrong DA prediction scores.

### ISSUE THREAD

**Issue by i-wanna-to** [TharinduDR/TransQuest#5]
I got wrong results after running the script **'examples.wmt_2020.en_de.trans_quest'**. I got the **predictions.txt** file in **temp/data/**. But I found that all DA prediction scores are distributed around 0.0041. I submitted it to CodaLab, the Pearson correlation is 0.094.  I don't know if the problem is in the code or the original data. Can you take a moment to help me look at this problem? Thank you very much!

Tom

**Comment by i-wanna-to** [comment]
Some of the predicted scores are as follows：
![prediction](https://user-images.githubusercontent.com/39059793/92066814-66660380-edd5-11ea-931e-4232344420eb.PNG)

**Comment by TharinduDR** [comment]
Can you share the configurations? It looks like the model has not been trained properly.

**Comment by i-wanna-to** [comment]
The configuration is as follows：

`from multiprocessing import cpu_count

SEED = 777
TEMP_DIRECTORY = "temp/data"
RESULT_FILE = "**DA-en-de-result.tsv**"
SUBMISSION_FILE = "**DA-en-de-predictions.txt**"
RESULT_IMAGE = "**DA-en-de-result.jpg**"
GOOGLE_DRIVE = False
DRIVE_FILE_ID = None
MODEL_TYPE = "xlmroberta"
MODEL_NAME = "xlm-roberta-large"

transformer_config = {
    'output_dir': 'temp/outputs/',
    "best_model_dir": "temp/outputs/best_model",
    'cache_dir': 'temp/cache_dir/',

    'fp16': False,
    'fp16_opt_level': 'O1',
    'max_seq_length': 128,
    'train_batch_size': 8,
    'gradient_accumulation_steps': 1,
    'eval_batch_size': 8,
    'num_train_epochs': 3,
    'weight_decay': 0,
    'learning_rate': 2e-5,
    'adam_epsilon': 1e-8,
    'warmup_ratio': 0.1,
    'warmup_steps': 0,
    'max_grad_norm': 1.0,
    'do_lower_case': False,

    'logging_steps': **30000**,
    'save_steps': **30000**,
    "no_cache": False,
    "no_save": False,
    "save_recent_only": True,
    'save_model_every_epoch': False,
    'n_fold': 3,
    'evaluate_during_training': False,
    "evaluate_during_training_silent": True,
    'evaluate_during_training_steps': **30000**,
    "evaluate_during_training_verbose": True,
    'use_cached_eval_features': False,
    "save_best_model": True,
    'save_eval_checkpoints': True,
    'tensorboard_dir': None,
    "save_optimizer_and_scheduler": True,

    'regression': True,

    'overwrite_output_dir': True,
    'reprocess_input_data': True,

    'process_count': cpu_count() - 2 if cpu_count() > 2 else 1,
    'n_gpu': 1,
    'use_multiprocessing': True,
    "multiprocessing_chunksize": 500,
    'silent': False,

    'wandb_project': None,
    'wandb_kwargs': {},

    "use_early_stopping": True,
    "early_stopping_patience": 10,
    "early_stopping_delta": 0,
    "early_stopping_metric": "eval_loss",
    "early_stopping_metric_minimize": True,
    "early_stopping_consider_epochs": False,

    "manual_seed": SEED,

    "config": {},
    "local_rank": -1,
    "encoding": None,
}
`

Bold fonts represent the place I modified. Other configurations remain the same as yours. I did not modify the code in ‘examples.wmt_2020.en_de.trans_quest’. 

Thank you so much for taking the time to help me solve this problem. Thanks again!

**Comment by TharinduDR** [comment]
I suspect that it is because you have changed the logging_steps, save_steps and evaluate_during_training_steps to a larger value and the best model does not get saved properly. Try again with smaller values such as 100, 200

**Comment by i-wanna-to** [comment]
OK, thank you again for your kind help!

**Comment by i-wanna-to** [comment]
I followed the configuration provided in your source code to experiment. But the predicted DA score is still the same. All prediction scores are distributed around 0.0041. I also conducted experiments in other language pairs, and none of them encountered similar problems. I wonder if there is a problem with the source code in 'examples.wmt_2020.en_de.trans_quest'.

**Comment by TharinduDR** [comment]
I will try to reproduce the error from my side

**Comment by i-wanna-to** [comment]
That's fine. Thank you! I will wait.

**Comment by TharinduDR** [comment]
There was a problem in the configurations. I corrected it, please run it again with this configuration - 

`'evaluate_during_training': True,`

**Comment by i-wanna-to** [comment]
Okay, I will run it again. Thank you.
